src/etc/inc/auth.inc

```diff
@@ -474,6 +474,8 @@ function local_user_set(& $user) {
 		$user_shell = "/bin/tcsh";
 	} elseif (userHasPrivilege($user, "user-copy-files")) {
 		$user_shell = "/usr/local/bin/scponly";
+	} elseif (userHasPrivilege($user, "user-copy-files-chroot")) {
```

Comment:
Validation is probably a lost cause there since it could be groups that conflict. Better to warn them and let it go and, like this, in cases of conflict have a good default behavior between the two. There's a case to be made either way. Group with the strict permission and a user with the looser one to override that way, but given the choice I'd prefer to go with the stricter choice when there is a conflict.